src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -0,0 +1,8 @@
+const temporaryBattleList: { [battleName: string]: TemporaryBattle } = {};
```

Comment:
Can we capitalize this as it's a global variable.
```suggestion
const TemporaryBattleList: { [battleName: string]: TemporaryBattle } = {};
```